webextension/js/popup.js

```diff
@@ -516,13 +516,27 @@ Logic.registerPanel(P_CONTAINERS_LIST, {
           previousElement.focus();
         }
       }
+      function select(num) {
+        const element = selectables[num];
+        if (element) {
+          element.click();
+        }
+      }
       switch (e.keyCode) {
       case 40:
         next();
         break;
       case 38:
         previous();
         break;
+      case 48:
```

Comment:
actually let's remove this case for `10` too; `0` as `10` may confuse folks.

Reply:
Done. But I'll be sad to see my tenth shortcut key leave :(